content/geometry/General.adoc

```diff
@@ -4,6 +4,14 @@
 The unit used for object dimensions is meters.
 
 == Coordinate systems
+
+=== Introduction to coordinate systems
+[tbd]
+
+=== Global coordinate systems
+[tbd]
+
+=== Local coordinate systems
```

Comment:
The contents of this would fit better in the introduction I think. Because these are general definitions true for all used coordinate systems. And the rotation order is missing.

Reply:
@LudwigFriedmann can you weigh in on how you would like to structure this?

Reply:
Hi @ClemensLinnhoff , if we have the need to specify coordinate frames for the material subgroup, as well (e.g. to definde BRDF angles), we could also move this to the subgroup-overarching "Introduction" chapter. Otherwise, "Geometry -> General -> Coordinate Systems" should be the right place.